webapp/src/components/root.jsx

```diff
@@ -7,7 +7,20 @@ export default class Root extends React.PureComponent {
         return (
             <React.Fragment>
                 <ConfirmFeedbackModal/>
+                <style dangerouslySetInnerHTML={{__html: injectedCSS}}/>
             </React.Fragment>
         );
     }
-}
\ No newline at end of file
+}
+
+const injectedCSS = `
+.sidebar-item[href$="@surveybot"] .icon__bot::before {
```

Comment:
neat trick, but personally I think this could morph into a maintenance nightmare :) i've seen it happen before. Anytime `.sidebar-item` is updated or styling is changed around the `sidebar` component, it will break this plugin, in some way. Unless `.sidebar-item` is something this plugin owns.

I know there is an ongoing discussion if you guys require this feature or not. If this is absolutely necessary I would recommend creating a style/theme api to inject assets for your webapp plugin, rather than doing this.

Reply:
Yeah, I knew this was a hack when I submitted it, and we've decided to cut this behaviour until we have a proper way for bot accounts to define their own sidebar icon.

I definitely felt clever about it, but I think that's usually a bad sign 😛